### ISSUE TITLE
Change tabIndex default to off?

### ISSUE THREAD

**Issue by iangilman** [openseadragon/openseadragon#874]
Currently, we default to giving our canvas a tabIndex, which allows the user to tab into it. Unfortunately, on Chrome this also means that the page auto-scrolls to our viewer. We've provided a mechanism for turning that off (#805), but maybe it should be off by default? On the other hand, that would make the viewer less accessible by default. 

Of course, if there is a way to allow tabbing but disable the auto-scroll that would be ideal!

**Comment by iangilman** [comment]
Hmm, looks like we didn't document how to disable it... at the very least the docs for the tabIndex option should include that passing `""` for it disables tabbing.

**Comment by jrochkind** [comment]
I'd like a way to set the tabindex to whatever I want, within the page I put it in, rather than just turn it off. Possible?

**Comment by iangilman** [comment]
@jrochkind Yep, that should work! Note that it's `tabIndex` not `tabindex` (case difference).
